crates/sui-framework/packages/sui-system/sources/voting_power.move

```diff
@@ -51,7 +51,7 @@ module sui_system::voting_power {
         let (info_list, remaining_power) = init_voting_power_info(validators, threshold);
         adjust_voting_power(&mut info_list, threshold, remaining_power);
         update_voting_power(validators, info_list);
-        check_invariants(validators);
+        // check_invariants(validators);
```

Comment:
Can we add a comment with some context here -- (how temporary is the commenting out, why is it here, etc)?